packages/vat-data/README.md

```diff
@@ -0,0 +1,17 @@
+# Agoric Vat Data
+
+This package provides access to the Vat Data facility.
+
+## Tips
+
+### Synchronous makers
+
+The durable kind maker functions are synchronous. When converting a maker that is async, you'll have to ensure that all necessary data is already available and need not be awaited in the `prepare*`.
+
+The reason for this constraint is that *all prepares happen in the first crank* of the event loop.
```

Comment:
All *reviving* prepares must happen in the first crank. All prepares that make only new ones can, and often must, happen after the first crank. While a prepare that is known to only be making a new one could have been manually written as a `makeDurable*` or `defineDurable*`, usually `prepare*` is the better style for writing them anyway. We don't want to discourage use of `prepare*` for fresh creation after the first crank.

Reply:
On this one, I'm not sure if my comment is due to the same confusion or not.

@turadg , the original text here refers to "the `prepare*`". The "the" makes sense if it is talking about the `prepare` function exported as entry point to the contract. But the "`*`" makes sense for the behavior rebinding functions, so I'm not sure which you mean. Then "*all prepares*" sounds like it is not talking about the exported `prepare` function, since there is only one per contract.

Now that we understand how I was confused, please let me know if my comment above is relevant.